test/clj/rems/api/test_applications.clj

```diff
@@ -955,12 +955,31 @@
                                                    {:form form-id :field "req1" :value "req"}
                                                    {:form form-id2 :field "opt2" :value "opt"}
                                                    {:form form-id2 :field "req2" :value "req"}]}))))
-    (testing "can't set value of text field to JSON"
-      (is (= {:success false
-              :errors [{:form-id form-id :field-id "req1" :type "t.form.validation/invalid-value"}]}
+    (testing "validation for set value of text field to JSON"
+      (is (= {:success true
+              :warnings [{:form-id form-id :field-id "req1" :type "t.form.validation/invalid-value"}]}
              (send-command user-id {:type :application.command/save-draft
                                     :application-id app-id
                                     :field-values [{:form form-id :field "req1" :value [[{:column "foo" :value "bar"}]]}]}))))
+    (testing "column name validation for table fields"
```

Comment:
this `(testing)` was moved up from below, because allowing save draft to go through with validation warnings causes application state to change. previously save draft wouldn't add event to application events stack with validation errors.